docs/world api.md

```diff
@@ -680,3 +680,60 @@ def generate_output(self, output_directory: str):
     generate_mod(src, out_file, data)
 ```
 
+### Documentation
+
+Every World implemented should have a tutorial and a game info page. These are both rendered on the website by reading
+the `.md` files in your world's `/docs` directory.
+
+#### Game Info
+The game info page is for a short breakdown of what your game is and how it works in Archipelago. Any additional
+information that may be useful for the player to know when learning your randomizer should also go here. The file format
+is `<language key>_<game name>.md`. While you can write these docs for multiple languages and can tell the `WebWorld`
+you have multiple of these docs, they currently do not render on the website.
+
+#### Tutorials
+Your game can have as many tutorials in as many languages as you like, with each one having a relevant `Tutorial`
+defined in the `WebWorld`. The file name you use aren't particularly important, but it should be descriptive of what
+the tutorial is covering, and the name of the file must match the url extension provided in the `Tutorial`.
+
+### Tests
+
+Ever World is expected to write unit tests that cover its logic, to ensure no logic bug regressions. This can be done
```

Comment:
I believe "each" is more correct. Also we use lowercase "world" in other places.
```suggestion
Each world is expected to include unit tests that cover its logic, to ensure no logic bug regressions. This can be done
```